src/index.js

```diff
@@ -48,42 +42,76 @@ class CarRacing {
     }
     getElement(ID.RACING_COUNT_INPUT).focus();
     this.cars = carNames.map(name => new Car(name));
+
+    changeActivity([ID.CAR_NAMES_INPUT, ID.CAR_NAMES_SUBMIT]);
   }
 
   onSubmitRacingCount(count) {
-    if (this.winners.length) {
-      return alert(MESSAGE.REINPUT_COUNT);
-    }
     if (!this.cars.length) {
       return alert(MESSAGE.NO_CAR);
     }
     if (!validateRacingCount(count)) {
       return alert(MESSAGE.WRONG_COUNT);
     }
-    getElement(ID.RACING_COUNT_INPUT).blur();
-    moveCars(this.cars, count);
-    getElement(ID.RACING_STATUS).insertAdjacentHTML('afterbegin', this.printResult());
-    getElement(ID.RACING_WINNERS).insertAdjacentHTML('afterbegin', winnersView(this.getWinner()));
+
+    changeActivity([ID.RACING_COUNT_INPUT, ID.RACING_COUNT_SUBMIT]);
+    this.progressGame(parseInt(count))
   }
 
   onClickRestart() {
     getElement(ID.CAR_NAMES_INPUT).value = '';
     getElement(ID.RACING_COUNT_INPUT).value = '';
     removeAllChildNodes(getElement(ID.RACING_WINNERS));
     removeAllChildNodes(getElement(ID.RACING_STATUS));
+    changeActivity([ID.CAR_NAMES_INPUT, ID.CAR_NAMES_SUBMIT, ID.RACING_COUNT_INPUT, ID.RACING_COUNT_SUBMIT]);
+    getElement(ID.RESTART_BUTTON).style.visibility="hidden";
     this.cars = [];
-    this.winners = [];
   }
 
-  printResult() {
-    return this.cars.map(car => userMovementView(car)).join('');
+  progressGame(count){
+    let startTime = 0, progressCount = 0, progressSecond = 0;
+    initRacingStatus(this.cars);
+
+    const render = (timeStamp) => {
+      if(!startTime) startTime = timeStamp;
+      progressSecond = Math.floor((timeStamp - startTime)/1000);
+      if(progressCount === count){
+        cancelAnimationFrame(render);
+        clearLoadingView(this.cars);
+        this.findWinner(this.cars);
+        resultView(this.winners);
+        return this.delayResult(this.winners);
+      }
+      if(progressCount!==progressSecond){
```

Comment:
띄어쓰기를 일관성있게 유지해는 것이 좋을 것 같습니다.
prettier에 대한 설정은 되어있는 것 같은데 동작을 안하는 것 같습니다. 확인 부탁드려요!
```suggestion
      if(progressCount !== progressSecond){
```